pkg/apiserver/apiserver.go

```diff
@@ -40,7 +40,8 @@ var (
 	Scheme = runtime.NewScheme()
 	// Codecs provides methods for retrieving codecs and serializers for specific
 	// versions and content types.
-	Codecs = serializer.NewCodecFactory(Scheme)
+	Codecs    = serializer.NewCodecFactory(Scheme)
+	TokenPath = "/var/run/antrea/apiserver/controller-loopback-client-token"
```

Comment:
Maybe no need to have controller/agent prefix for the token as this is only about apiserver storing its privileged token, not related to the apis they are serving.

Reply:
Updated. I used to wrongly assumed both the agent and the controller mounted `/var/run/antrea` and thus try to avoid the same name.

Reply:
I forgot this, you are right about the shared path. But since controller doesn't mount that path yet, we can keep it as is now, we can find solution later if both of them mount it.